app/src/main/java/com/example/happymeals/ingredient/Ingredient.java

```diff
@@ -160,4 +165,8 @@ public void setUnit( String unit ) {
     public void setCategory( String category ) {
         this.category = category;
     }
+
+    public void setNeedsUpdate(Boolean update) {
```

Comment:
What does this function do? What is the purpose up the update. Can you comment the method?